leapp/exceptions.py

```diff
@@ -130,3 +130,10 @@ def __init__(self, message, severity=ErrorSeverity.ERROR, details=None):
         super(StopActorExecutionError, self).__init__(message)
         self.severity = severity
         self.details = details
+
+
+class RequestStopAfterPhase(LeappError):
+    """
+    This exception is used to gracefully stop the current actor and request the stop of the workflow execution after
+    the current phase.
+    """
```

Comment:
When raised in https://github.com/oamg/leapp-repository/pull/475:
`TypeError: __init__() takes exactly 2 arguments (1 given)`
```suggestion
    """
    def __init__(self):
        super(RequestStopAfterPhase, self).__init__("nigga stop please")
```